RecoPPS/Local/src/RPixRoadFinder.cc

```diff
@@ -69,14 +66,19 @@ void RPixRoadFinder::findPattern(bool isBadPot) {
 
       math::Error<3>::type globalError = ROOT::Math::SimilarityT(theRotationTMatrix, localError);
 
-      // create new collection for planes 0 and 5 of pot 45-220-fr
-
-      if (isBadPot == true && myid.arm() == 0 && myid.station() == 2 && localV.x() > 0 &&
-          (myid.plane() == 0 || myid.plane() == 5)) {  // 45-220-far
+      // create new collections for bad and good pots
 
-        temp_all_hits_badPot.emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
+      if (isBadPot[0] == true && myid.arm() == 0 && myid.station() == 2) {  // 45-220
+        temp_all_hits_badPot[0].emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
+      } else if (isBadPot[1] == true && myid.arm() == 0 && myid.station() == 0) {  // 45-210
+        temp_all_hits_badPot[1].emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
+      } else if (isBadPot[2] == true && myid.arm() == 1 && myid.station() == 0) {  // 56-210
+        temp_all_hits_badPot[2].emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
+      } else if (isBadPot[3] == true && myid.arm() == 1 && myid.station() == 2) {  // 56-220
+        temp_all_hits_badPot[2].emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
```

Comment:
Shouldn't this be
```suggestion
        temp_all_hits_badPot[3].emplace_back(PointInPlane{globalV, globalError, it_rh, myid});
```
?

Reply:
Indeed! Thanks Andrea for spotting it... Fix committed.